### ISSUE TITLE
Is there a way to extract geographical data, i.e. long & lat in this model?

### ISSUE THREAD

**Issue by isam007** [Banus/crism_ml#1]
Is there a way to also extract geographical data, i.e. long & lat using this model? Where is long & lat data hidden? Neither .lbl or .hdr files seem to have it and .img file seem to have 438 bands. Thanks!

**Comment by Banus** [comment]
No, geographical data is part of metadata and they are not available in the HDR or LBL files. They are however available on the CRISM website. For the example image is this:
http://crism-map.jhuapl.edu/details.php?data=hrl_webmap_polygons&shape=5308&x=-165.07382&y=-29.10805

And you can see the coordinates both in the URL and in the summary table.

To find the page for a given file you need to extract the first part of the code for the file name, for example for `hrl00016cfe_07_if181l_trr3` you need to select `00016cfe`, and search it in the [CRISM map](http://crism-map.jhuapl.edu/).

The CRISM files have indeed 438 bands, but we select a subset. See the [crism_to_mat](https://github.com/Banus/crism_ml/blob/8aa4623bdc5ae4ba8da295af54a9ad7f810cc81b/crism_ml/io.py#L118) function to see what kind of format is expected by our toolbox.
